src/app/common/search-input/search-input.component.scss

```diff
@@ -10,3 +12,12 @@
 	float: right;
 	z-index: 500;
 }
+
+.min-count {
+	color: map-get($palette-gray, 700);
+	font-size: 14px;
+
+	div {
+		padding-top: 5px;
```

Comment:
Should this be using the `pt-5` class in the template instead? We have a few different ways of providing these spacers.